.gitignore

```diff
@@ -14,3 +14,4 @@
 
 Cargo.lock
 build/
+.gitlongtag
```

Comment:
I feel like `.gitlongtag` should be in the `build/` directory, as it is a build artifact. Then it wouldn't need its own `.gitignore` rule.

Reply:
Yep.  In new push to this PR.